CHANGES.md

```diff
@@ -7,6 +7,7 @@ Next release
 
 Features
 --------
+* [#569](https://github.com/java-native-access/jna/pull/569): Added PRINTER_INFO_2 support and GetPrinter function [@IvanRF](https://github.com/IvanRF)
```

Comment:
Please make this look like the lines below. The class names are usually included with the full name like `com.sun...`, not everybody uses Windows.